charts/alfresco-repository/templates/ingress.yaml

```diff
@@ -1,10 +1,12 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "alfresco-repository.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
-  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
-  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
-  {{- end }}
+{{/*
+We only support nginx ingress for now: https://alfresco.atlassian.net/browse/OPSEXP-131
+*/}}
+{{- if not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
+  {{- $_ := unset .Values.ingress.annotations "kubernetes.io/ingress.class" }}
+  {{- $_ = set .Values.ingress.annotations "kubernetes.io/ingress.class" "nginx" }}
```

Comment:
it's hard to understand what we are achieving here, can you describe it?

also adding some unit test would help

Reply:
we're overriding a potential ingress.class annotation by unsetting it regardless of its value and setting it again to the nginx value